kde-frameworks/prison/prison-5.109.0-r1.ebuild

```diff
@@ -11,23 +11,27 @@ HOMEPAGE="https://invent.kde.org/frameworks/prison"
 
 LICENSE="GPL-2"
 KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~x86"
-IUSE="qml"
+IUSE="+dtmx qml +video +zxing"
```

Comment:
Avoiding these seems reasonable, especially in the case of zxing.

Reply:
How will `prison` revdeps react if none of these are available?